.github/gendocs/main.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"log"
+	"os"
+
+	"github.com/plexsystems/konstraint/internal/commands"
+	"github.com/spf13/cobra/doc"
+)
+
+func main() {
+	cliDocsDir := "./docs/cli"
+
+	if _, err := os.Stat(cliDocsDir); os.IsNotExist(err) {
```

Comment:
disclaimer, incoming opinion: Not usually a fan of creating directories. Especially in this case where once the directory is initially created, it should never be deleted. So the vast majority of this code goes unused and detracts from the real work going on here. Does it matter a ton? Not really

Reply:
Agreed, I added a .gitkeep so I can check in the empty dir.